modes/tr/is-ilani.md

```
@@ -0,0 +1,247 @@
+
+---
+
+## Çıkarılan Anahtar Kelimeler
+(ATS optimizasyonu için ilandan 15-20 anahtar kelime)
+```
+
+### 2. Takipçiye Kaydet
+
+**Yeni** kayıt için `data/applications.md`'yi doğrudan düzenleme. Bunun yerine `batch/tracker-additions/{num}-{sirket-slug}.tsv` dosyasına tek satır TSV yaz (8 veya 9 sekme ile ayrılmış sütun):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)